index.html

```diff
@@ -52,6 +52,25 @@ <h2>I am a Software Developer</h2>
                 <li><a href="https://medium.com/"><img src="./media/icons/Medium.svg" alt="medium icon"></a></li>
             </ul>
         </section>
+        <div class="pop-up hidden">
+            <div id="pop-up-head">
+                <h2>Multi-Post Stories</h2><span>&#x2715;</span>
+            </div>
+            <ul>
+            </ul>
+            <div id="pop-up-body">
+                <img id="pop-up-img" src="./media/Img-Placeholder.jpg" alt="image containing a dancer">
+                <div id="pop-up-text">
+                    <p>
+                        Lorem 
+                    </p>
+                    <div class="buttons">
+                        <a id="demo" href="#">Live Demo<img src="./media/icons/see-live.svg" alt="icon"></a>
+                        <a id="src" href="#">See Source<img src="./media/icons/github-white.svg" alt="icon"></a>
+                    </div>
+                </div>
+            </div>
+        </div>
         <section id="works">
```

Comment:
- [ ] @Peter1907 and @Mov305, I think you didn't understand the changes requested, instead of hard coding this section of projects, you should load the projects dynamically using javascript as you did for the popup, using the array already created, this means the already hardcoded projects need to be removed.